t/22-dashboard.t

```diff
@@ -40,6 +40,10 @@ is_deeply(\@h2, ['opensuse', 'opensuse test'], 'two groups shown (from fixtures)
 my $test_parent = $parent_groups->create({name => 'Test parent', sort_order => 2});
 
 subtest 'Validation errors' => sub {
+    $t->get_ok('/dashboard_build_results/?group=valid-regex')->status_is(200);
```

Comment:
Maybe it would be good to actually test that we get results back.
```suggestion
    $t->get_ok('/dashboard_build_results/?group=opensuse test')->status_is(200)->content_like(qr{/group_overview/1002});
```